openpype/hosts/fusion/api/lib.py

```diff
@@ -309,6 +309,12 @@ def get_fusion_module():
     return fusion
 
 
+def get_bmd_library():
+    """Get bmd library"""
+    bmd = getattr(sys.modules["__main__"], "bmd", None)
```

Comment:
Why not return it directly?

Reply:
I could but I kept the style of the other functions in there like `get_fusion_module` and `get_current_comp`
I could fix those also if you want?

Reply:
I'm fine with refactoring both, also fine with what it is now and moving 'cosmetics' to another PR to keep things simple. Feel free to mark resolved.